### PR TITLE
Control page/examples

### PR DESCRIPTION
**Pull request by britt6612**
Related Issue: #416 

Work Included: 
* Implemented the controls page examples to use the usage examples
* Each example will now have the more to view the code and other props

Screen Shots 
Screen Shot 2020-02-25 at 12 28 11 PM

### CODE REVIEW

File: aries-site/src/pages/components/controls.js [hunk 1]
```diff
@@ -62,6 +67,9 @@ const Controls = () => (
     </ContentSection>
     <ContentSection>
       <Subsection name="Button">
+      <SubsectionText>
```
Comment (taysea) [inline aries-site/src/pages/components/controls.js]: This comment applies to all the examples on this page, but I think there had been discussion a month or so ago (before you joined on the project) where we decided the content order would go:
1. Section title
2. Example
3. SubsectionText

I realize that isn't updated in the designs but came out of discussion, and I forgot about that when we talked this morning. For all of these, can you keep the order as listed above?

Comment (halocline) [reply aries-site/src/pages/components/controls.js]: @taysea This order has had additional discussion / rationale. Offline discussions with Greg, plus finding the Slack convo for reference. It starts here + next 5-6 comments: https://hpe.slack.com/archives/GPPLUK6LR/p1581632692012300

Comment (taysea) [reply aries-site/src/pages/components/controls.js]: Gotcha! Thanks for pointing me to that. I missed that conversation. PR looks good to me then.

File: aries-site/src/pages/components/controls.js [hunk 2]
```diff
@@ -70,29 +78,36 @@ const Controls = () => (
...
-        </SubsectionText>
       </Subsection>
     </ContentSection>
     <ContentSection>
       <Subsection name="Menu">
-        <MenuExample />
-        <SubsectionText>
+      <SubsectionText>
           Menu is used to filter or sort content on a page. It is similar to the
           select component. However, use the select component when the user must
```
Comment (halocline) [inline aries-site/src/pages/components/controls.js]: Can we add an `<Anchor />` around "select component" and link it to the Select component deep link? Also capitalize Select in both instances as select is the name of the component and thus a proper noun which should be capitalized.

File: aries-site/src/examples/components/controls/MenuExample.js [hunk 3]
```diff
@@ -13,23 +10,9 @@ export const MenuExample = () => {
...
   return (
-    <UsageExample
-      pad={{
-        horizontal: 'large',
-        vertical: 'large',
-        small: { horizontal: 'large', vertical: 'xlarge' },
-      }}
-    >
-      <Box background={background} round="xsmall">
+      <Box round="xsmall">
```
Comment (halocline) [inline aries-site/src/examples/components/controls/MenuExample.js]: good catch

File: aries-site/src/pages/components/controls.js (hunk 2 above)
Comment (britt6612) [reply aries-site/src/pages/components/controls.js]: good thinking! I added that in

**Pull request merged**